.github/workflows/ansible-test-sanity.yml

```diff
@@ -39,6 +39,7 @@ jobs:
         - stable-2.12
         - stable-2.13
         - devel
+        - stable-2.13.0rc1
```

Comment:
It looks like this doesn't work with "ansible-test-gh-action":
```
Run echo ::group::Installing ansible-core version stable-2.13.0rc1...
Installing ansible-core version stable-2.13.0rc1...
Run set -x ; python -m pip install https://github.com/ansible/ansible/archive/stable-2.13.0rc1.tar.gz --disable-pip-version-check --user ; set +x
+ python -m pip install https://github.com/ansible/ansible/archive/stable-2.13.0rc1.tar.gz --disable-pip-version-check --user
Collecting https://github.com/ansible/ansible/archive/stable-2.13.0rc1.tar.gz
  ERROR: HTTP error 404 while getting https://github.com/ansible/ansible/archive/stable-2.13.0rc1.tar.gz
ERROR: Could not install requirement https://github.com/ansible/ansible/archive/stable-2.13.0rc1.tar.gz because of HTTP error 404 Client Error: Not Found for url: https://codeload.github.com/ansible/ansible/tar.gz/stable-2.13.0rc1 for URL https://github.com/ansible/ansible/archive/stable-2.13.0rc1.tar.gz
Error: Process completed with exit code 1.
```

The existing stable-2.13 should be close enough to reproduce the issues, I think.